config/packages/twig.yaml

```diff
@@ -4,3 +4,4 @@ twig:
     globals:
         build_tag: '%env(IMAGE_BUILD_TAG)%'
         slot_interval: '%app.slot_interval%'
+        skills_to_display: ['ch_vpsp', 'ch_vl']
```

Comment:
this value must be stored near the other ones : `$importantSkillsToDisplayLimit: "%app.most_important_skills_to_display%"`